### ISSUE TITLE
The browser console will output non-unique id error

### ISSUE THREAD

**Issue by lanmolsz** [mulesoft/api-console#551]
The errors caused by these components：
- `auth-methods`
    - `auth-method-basic.html` line 67
- `paper-input`
    - `paper-input.html` line 125
- `raml-request-url-editor`
    - `raml-request-url-editor.html` line 163

Can solve it?

**Comment by jarrodek** [comment]
This is Polymer's paper-input issue. Others can be fixed.

**Comment by lanmolsz** [comment]
@jarrodek Because the parameters are repeated

**Comment by jarrodek** [comment]
Even in ARC elements the error is caused by Polymer's `iron-input` element. The best workaround is to use the following code on the page source (before loading elements and polyfills):

```javascript
<script>
    /* this script must run before Polymer is imported */
    window.Polymer = {
      dom: 'shadow'
    };
  </script>
```
This will force Chrome to use shadow DOM which encapsulate the elements and therefore ID conflicts won't happen.

I am not sure if Chrome team is planning to update 1.x.x version of the `iron-input`. I am planing to move the elements to version 3.x.x once it's released so probably it will be fixed there.

**Comment by jarrodek** [comment]
There's a plan to upgrade the elements to Polymer v3 which should not have this issue in the future.
However upgrade will take a time (weeks probably) so it's nothing I can fix fast.

**Comment by jarrodek** [comment]
This issue is fixed in upcoming 6.0.0 release
